src/main.ts

```diff
@@ -29,24 +29,24 @@ const map = new WebScene({
 });
 
 const view = new SceneView({
-  container: "viewDiv",
+  container: "view-div",
   map,
   qualityProfile: "high",
 });
 
-map.when().then(() => {
+void map.when().then(() => {
```

Comment:
Alternatively we can add an ignore or a `.catch` to these promises.

Semantically `void` is probably the cleanest thing to do but it's also a pretty unusual syntax so it might be better to do one of the other options...

Reply:
I like `void` and it also makes sense as a linter rule. As pointed out in the comment however, it would be good to have them reported as errors instead of warnings.